packages/translate/lib/startup.ts

```diff
@@ -0,0 +1,118 @@
+import {Translation} from './types';
+import {NOT_STARTED_REASON, NotStartedReason} from './constants';
+import t from './texts';
+
+/**
+ * Translate the reason for test not being executed
+ */
+export function translateStartupError(code: NotStartedReason): Translation | undefined {
```

Comment:
Is this function should be renambed since we used "NotStartedReason"-like names?